_sass/variables.scss

```diff
@@ -2,6 +2,9 @@
 $large-breakpoint: 64em !default;
 $medium-breakpoint: 42em !default;
 
+//Navbar color (cell-phones)
+$theme_color: #157878 !default;
```

Comment:
Well :) you surely know a lot more about Jekyll and ruby than I do :) maybe then it should consider removing that tag from the template, it can be hard to find if someone (like me :p ) tries to change all the variables and doesn't know about that one

thanks anyway :)